### PR TITLE
Limit the regcode input field

### PR DESCRIPTION
**Pull request by dgdavid**
Related to https://bugzilla.suse.com/show_bug.cgi?id=1098576

### CODE REVIEW

File: package/yast2-registration.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Sep  4 08:34:31 UTC 2018 - dgonzalez@suse.com
+
+- Limit max length of regcode to 512 (bsc#1098576)
```
Comment (mchf) [inline package/yast2-registration.changes]: 512 of "what" ;-)

File: src/lib/registration/ui/base_system_registration_dialog.rb [hunk 2]
```diff
@@ -65,6 +65,9 @@ def run
           Yast::GetInstArgs.enable_next || Yast::Mode.normal
         )
 
+        # Limit the reg_code InputField to 512 (bsc#1098576)
```
Comment (mchf) [inline src/lib/registration/ui/base_system_registration_dialog.rb]: well ... a bit useless comment ;-)

**Comment by mchf** [comment]
@lslezak does it make sense to also limit min length when already touching the code?

File: src/lib/registration/ui/base_system_registration_dialog.rb (hunk 2 above)
Comment (dgdavid) [reply src/lib/registration/ui/base_system_registration_dialog.rb]: Actually, it "explains" why that control was introduced, linking to the associated bug. As far as I have understood until now (reading the code), this is a quite common practice in YaST. Isn't it?

If you want, I could reword it to something like `Avoid sending too large request limiting the registration code (bsc#1098576)` or whatever other reasonable suggestion :)

Comment (lslezak) [reply src/lib/registration/ui/base_system_registration_dialog.rb]: I'm fine with the current comment, the most important part is the bug number which is present.

**Comment by lslezak** [comment]
:heavy_check_mark: Public Jenkins [job #9](http://ci.opensuse.org/job/yast-yast-registration-master/9/) successfully finished
